extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/InitTaskProcessor.java

```diff
@@ -20,40 +21,47 @@
 
 public class InitTaskProcessor {
 
+    private static final String INIT_CONTAINER_WAITER_NAME = "init";
+    private static final String INIT_CONTAINER_WAITER_DEFAULT_IMAGE = "groundnuty/k8s-wait-for:no-root-v1.7";
```

Comment:
Never heard about it.... We need to have a deeper look. 

The fact that it uses Alpine is not a great start...

Reply:
Since this image seems to be commonly used by the community for this purpose and also it was already used before these changes, I think the way to proceed is to open a new issue to investigate how secure it is and the alternatives.

@cescoffier @gsmet @iocanel do you think we can merge these changes that improve this existing feature or do you prefer put it hold off?

Reply:
I wouldn't hold off this pull request because of the image. We need to track it in a separate issue.

Reply:
What image was used before ? Yeah. We should not be using/proposing alpine unless absolutely no other option. 

What is the requirements for the image and why not use the base images we otherwise use here ? (That would mean 0 mb extra to fetch :)